learn-to-use-vim/README.md

```diff
@@ -0,0 +1,46 @@
+<h1 align="center"><a href="https://egghead.io/courses/learn-to-use-vim">Learn to use Vim</a></h1>
+<p align="center"><img src="https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/045/full/EGH_Vim.png" width="200></p>
```

Comment:
Took me a while to figure out why the About section wasn't showing up. You forgot the second " after the width in the img.

Reply:
You know what, I look at the preview and noticed that something looked funny, but I thought it was some extension that I installed on my vscode since it was complaining that I can't have HTML code in markdown haha

Thank you for spotting this!